### PR TITLE
Window Popup

### PR DESCRIPTION
**Pull request by mstanley-me**
# In this milestone I:

 - implement a popup window that includes the project details in both desktop and mobile.
 - I add some change to the mobile version 

Thanks for your review😁

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -103,9 +104,71 @@ <h4>Tonic</h4>
...
+                            han printer took a galley of
+                            type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it
+                            1960s with the releaLorem
+                            Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been
+                            the industry's standard dummy
+                            text ever since the 1500s, when an unknown printer took a galley of type and scrambled it
+                            1960s with the relea
+                        </p>
+                        <div class="pop-items">
+                            <ul class="languages">
```
Comment (Felipeg005) [inline index.html:157]: The project's design it's looking great 💯, but when you open the project popup in the mobile version and in the desktop version there are still some issues to be fixed as I show you in the next screenshots:

## Mobile
Your design:
image

Figma Design: 
image

In the mobile version, the popup isn't fit on the screen and I'm not able to close or access the live and source button, for very short screen sizes you may consider to add a scroll for the popup.

- [ ] Kindly fix the popup design to make it responsive to all screen sizes.

## Desktop

Your design: 
image


Figma design: 
image

In the desktop version, I'm facing the same issue.

- [ ] Kindly update your styles to follow the Figma designed and make it responsive.

File: index.html [hunk 2]
```diff
@@ -131,10 +194,11 @@ <h4>Multi-Post Stories</h4>
...
                     </ul>
-                    <button class="btn" type="submit">See Project</button>
+                    <button class="btn openPopup" type="submit">See Project</button>
                 </div>
                 <img class="work-pic-1" src="./images/desktop-images/img2.png" alt="professional art printing image" />
             </div>
+
             <div class="container">
                 <img class="work-pic" src="./images/Portoflio_Card/work2.png" alt="facebook 360 image" />
                 <img class="work-pic" src="./images/desktop-images/img3.png" alt="landing view, desktop-image" />
```
Comment (Felipeg005) [inline index.html:204]: Lastly but not least:

- [ ] Please add and acknowledgments section in your readme file giving credits for the design creator from Behance 😄

**Review by Felipeg005 — CHANGES_REQUESTED** [review]
Hi @mstanley-me, 

Good job so far 👏 !

## Highlights ✅ 
- Good Readme file
- Follows HTML good practices
- All linters passing

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mstanley-me** [comment]
Hello

**Comment by mstanley-me** [comment]
Can wwe meet now please!?

**Review by t-yanick — APPROVED** [review]
Hi @mstanley-me, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (mstanley-me) [reply index.html:190]: Thank you so much i'll do that!

**Pull request merged**